webtesting-react/src/App.test.js

```diff
@@ -0,0 +1,34 @@
+import React from 'react';
+import ReactDOM from 'react-dom';
+import { render, fireEvent } from 'react-testing-library';
+
+import App from './App';
+import Dashboard from './components/Dashboard';
+
+
+
+
+it('renders without crashing', () => {
+  const div = document.createElement('div');
+  ReactDOM.render(<App />, div);
+  ReactDOM.unmountComponentAtNode(div);
+});
+describe('<Dashboard />', () => {
+  
+  it('should render the action buttons strike, ball, foul, and hit', () => {
+    const { getByText } = render(<Dashboard />)
+    const button = getByText(/strike/i, /ball/i, /foul/i, /hit/i);
```

Comment:
you may want to select these individually so that you can test them by themselves. Right now, you aren't actually testing that these are working.